index.html

```diff
@@ -0,0 +1,186 @@
+					don’t hesitate to contact me.</p>
+			</section>
+			
+			<section class="icons-container">
+				<ul class="icons-containers">
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/></li>
+					<li><img src="./images/Icon - Linkedin.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Enabled.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Twitter.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Medium.png"/></li>	
+				</ul>				
+			</section>
+		</main>
+	</section>
+	
+	<section class="second_container">
+		<div class="header_title">
+		<h2>My Recent Works</h2>
+		<p class="line"></p>
+		</div>
+		<div class="recent_work_container">
+			<img src="./images/imgs.png" alt="works_section">
+			<div class="recent_work_container_text">
+				<h3>Multi-Post Stories</h3>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+			<div class="butons_container">
+				<button type="button">CSS</button>
+				<button type="button">HTML</button>
+				<button type="button">Boostrap</button>
+				<button type="button">Ruby</button>
+			</div>
+			<button  type="button" class="project_btn">See project</button>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<div class="box">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button  type="button"class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
```

Comment:
Please kindly implement the button in your styling: `Active`, `Hover`, and `Pressed` according to the project requirement

see design
![image](https://user-images.githubusercontent.com/78251927/229277799-1ec9dab3-294d-411f-a9dc-98604eaadc63.png)

Reply:
- [ ] Also, implementing the correct styles for all your buttons would be a nice ting that is capable of enhancing the user experience on your website. :wink: It will be be worth it :muscle: